http3/server.go

```diff
@@ -198,6 +198,12 @@ type Server struct {
 	// In that case, the stream type will not be set.
 	UniStreamHijacker func(StreamType, quic.ConnectionTracingID, quic.ReceiveStream, error) (hijacked bool)
 
+	// IdleTimeout specifies how long until idle clients connection should be
+	// closed. Quic PING frames are not considered activity for the purposes of
```

Comment:
Maybe generalize this statement, as not only PING frames reset QUIC's idle timeout? You could say that "idle" refers only to the HTTP/3 layer.